themes/default/content/blog/team-access-tokens/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Announcing Team Access Tokens for the Pulumi Service"
+
+date: 2022-08-216T14:00:00-07:00
+
+meta_desc: We are launching Team Access Tokens, which allow Organization and Team Admins to create access tokens scoped to a Pulumi Team.
+
+meta_image: meta.png
+
+authors:
+    - devon-grove
+    - meagan-cojocar
+
+tags:
+    - features
+    - pulumi-service
+---
+
+A few months ago we launched [Organization Access Tokens]({{< relref "/blog/organization-access-tokens" >}}) for the [Pulumi Service](https://app.pulumi.com) and saw overwhelmingly fast adoption from our customer base. Based on this customer demand, and existing customer feedback, we prioritized improvements in the scoping of access tokens. Today, we are launching Team Access Tokens, which allow Organization and Team Admins to create access tokens scoped to a [Pulumi Team]({{< relref "/docs/intro/pulumi-service/teams" >}}). Pulumi Service customers on the Enterprise and Business Critical editions can use Pulumi Teams to set role-based access controls (RBAC) for stacks by enabling Organization administrators to assign a set of stack permissions to a group of users.
+
+<!--more-->
+
+As a result of Pulumi Teams only being available on these editions, Team Access Tokens are also only available to customers on the Enterprise and Business Critical editions of the Pulumi Service.
```

Comment:
```suggestion
Since Pulumi Teams are only available on these editions, Team Access Tokens are also only available to customers on the Enterprise and Business Critical editions of the Pulumi Service.
```